app/src/main/java/mozilla/lockbox/presenter/RoutePresenter.kt

```diff
@@ -141,29 +86,7 @@ class RoutePresenter(
             .addTo(compositeDisposable)
     }
 
-    private fun showAutoLockSelections() {
-        val autoLockValues = Setting.AutoLockTime.values()
-        val items = autoLockValues.map { it.stringValue }.toTypedArray()
-
-        settingStore.autoLockTime.take(1)
-            .map { autoLockValues.indexOf(it) }
-            .flatMap {
-                AlertDialogHelper.showRadioAlertDialog(
-                    activity,
-                    R.string.auto_lock,
-                    items,
-                    it,
-                    negativeButtonTitle = R.string.cancel
-                )
-            }
-            .flatMapIterable {
-                listOf(RouteAction.InternalBack, SettingAction.AutoLockTime(autoLockValues[it]))
-            }
-            .subscribe(dispatcher::dispatch)
-            .addTo(compositeDisposable)
-    }
-
-    private fun showDialogFragment(dialogFragment: DialogFragment, destination: RouteAction.DialogFragment) {
+    fun showDialogFragment(dialogFragment: DialogFragment, destination: RouteAction.DialogFragment) {
         try {
             dialogFragment.setTargetFragment(currentFragment, 0)
```

Comment:
We need both versions of `showDialogFragment` (one specifically for autofill). Otherwise autofill will fail on this line.